app/controllers/listings_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ListingsController < ApplicationController
+
+  def new
+    @listing
```

Comment:
please do it like this
```@listing = Listing.new```

Reply:
We tried this but then our tests fail, since we haven't implemented the model for "Listing" - since that is being done in the other PR by Zane and Carla.. Realize that this PR might be a bit too "granular", by only focusing on creating a form. Will make them less granular going forward.